src/Core/src/Core/IGridLayout.cs

```diff
@@ -2,18 +2,57 @@
 
 namespace Microsoft.Maui
 {
+	/// <summary>
+	/// Represents a layout that arranges views in rows and columns.
+	/// </summary>
 	public interface IGridLayout : ILayout
 	{
+		/// <summary>
+		/// A GridRowDefinition collection for the GridLayout instance.
```

Comment:
All of these should probably use the interface names, e.g.:
`An IGridRowDefinition collection for the GridLayout instance.`

Reply:
Updated.